pyflakes/test/test_api.py

```diff
@@ -661,6 +668,9 @@ def runPyflakes(self, paths, stdin=None):
         if sys.version_info >= (3,):
             stdout = stdout.decode('utf-8')
             stderr = stderr.decode('utf-8')
+        # Workaround https://bitbucket.org/pypy/pypy/issues/2350
```

Comment:
looks like there will be a fix in the next release
https://bitbucket.org/pypy/pypy/commits/85ea4f3912ce